Sources/GRPC/ClientCalls/ClientCall.swift

```diff
@@ -131,6 +153,33 @@ extension StreamingRequestClientCall {
     }
   }
 
+  public func sendMessages<S: Sequence>(_ messages: S, flush: Bool = true) -> EventLoopFuture<Void> where S.Element == RequestMessage {
+    self.subchannel.flatMap { channel in
+      let writeFutures = messages.map { message in
+        channel.write(GRPCClientRequestPart.message(_Box(message)))
+      }
+      if flush {
+        channel.flush()
+      }
+      return EventLoopFuture.andAllSucceed(writeFutures, on: channel.eventLoop)
+    }
+  }
+
+  public func sendMessages<S: Sequence>(_ messages: S, promise: EventLoopPromise<Void>?, flush: Bool = true) where S.Element == RequestMessage {
+    if let promise = promise {
+      self.sendMessages(messages, flush: flush).cascade(to: promise)
+    } else {
+      self.subchannel.whenSuccess { channel in
```

Comment:
Could we replace this with e.g. `self.sendMessages(messages, flush: flush).then { *do nothing* }`?

Reply:
Why? Doing so would create an additional `messages.count` callbacks doing that.